powerauth-webauth-authentication/src/main/java/io/getlime/security/powerauth/lib/webauth/authentication/configuration/WebAuthServicesConfiguration.java

```diff
@@ -43,12 +47,28 @@
     private String nextstepServiceUrl;
 
     /**
+     * Whether invalid SSL certificates should be accepted.
+     */
+    @Value("${powerauth.service.ssl.acceptInvalidSslCertificate}")
+    private boolean acceptInvalidSslCertificate;
+
+    @Autowired
+    public WebAuthServicesConfiguration(SSLConfigurationService sslConfigurationService) {
+        this.sslConfigurationService = sslConfigurationService;
+    }
+
+    /**
      * Default credential store client.
      * @return Credential store client.
      */
     @Bean
     public CredentialStoreClient defaultCredentialStoreClient() {
-        return new CredentialStoreClient(credentialServerServiceUrl);
+        CredentialStoreClient client = new CredentialStoreClient(credentialServerServiceUrl);
+        // whether invalid SSL certificates should be accepted
+        if (acceptInvalidSslCertificate) {
```

Comment:
We are not disabling SSL validation for the client, but for all server application...